src/stdx.zig

```diff
@@ -1,3 +1,5 @@
+/// Extensions to the standard library -- things which could have been in std, but aren't.
```

Comment:
The `zig fmt` CI complains about this file. My guess is it wants this to be a module comment.
```suggestion
//! Extensions to the standard library -- things which could have been in std, but aren't.
```

Reply:
Argh, I was sure zig didn’t repeat Rust mistake of needing two types of the comments for this…